rag/svr/task_executor_refactor/dataset_wiki_generator.py

```
@@ -0,0 +1,778 @@
+        n_emb = len(embeddings) if embeddings is not None else 0
+    except TypeError:
+        n_emb = 0
+    if n_emb != len(pages):
+        logging.warning(
+            "artifact_persist: embedding count %d != pages %d for kb=%s; aborting",
+            n_emb, len(pages), kb_id_str,
+        )
+        return
+
+    rows: List[Dict] = []
+    for page, vec in zip(pages, embeddings):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)